verification/curator-service/api/openapi/openapi.yaml

```diff
@@ -834,15 +834,23 @@ paths:
           pattern: '^[a-f\d]{24}$'
     put:
       tags: [User]
-      summary: Updates a specific user's roles
+      summary: Updates a specific user. Can only update roles and hasSeenWelcomePopup.
```

Comment:
nit but I'd say this is implied by the arguments below

Reply:
I like to spell it out in English too, in case someone missed the additionalProperties tag or they're quickly scanning through the docs.